presto-verifier/src/test/java/com/facebook/presto/verifier/checksum/TestChecksumValidator.java

```diff
@@ -245,6 +275,75 @@ public void testFloatingPoint()
         assertEquals(mismatchedColumns.get(1).getMessage(), Optional.of("relative error: 1.0000500025000149E-4"));
     }
 
+    @Test
+    public void testFloatingPointArray()
```

Comment:
can you add a test for the empty table case too?

Reply:
Yes, I was going to, but hit that snag with nulls in the Map.
I can see if I can use that HashMap to generate the nulls.